utils/anahistory.py

```diff
@@ -0,0 +1,370 @@
+"""
+Tools to analyze the time evolution of detectors.
+"""
+
+import numpy as _np
+
+class ChannelTimeRange(object):
+    """Represents a range of scans in TimeSeriesData, for a given VFAT and
```

Comment:
Is `range` here the same meaning as it is in the documentation? e.g. a collection of "bad" scans?

Reply:
It's a set of consecutive scans, with no special meaning